core/src/main/java/org/elasticsearch/search/suggest/completion/context/CategoryQueryContext.java

```diff
@@ -118,7 +110,8 @@ public CategoryQueryContext build() {
         CATEGORY_PARSER.declareBoolean(Builder::setPrefix, new ParseField("prefix"));
```

Comment:
Can these ParseFields be linked to the String constants in CategoryContextMapping?